transport/src/main/java/io/netty/channel/AbstractCoalescingBufferQueue.java

```diff
@@ -156,6 +160,9 @@ public final ByteBuf remove(ByteBufAllocator alloc, int bytes, ChannelPromise ag
         }
         readableBytes -= originalBytes - bytes;
         assert readableBytes >= 0;
+
+        decrementPendingOutboundBytes(originalBytes - bytes);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Cast one of the operands of this substraction operation to a "long". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2184)

Reply:
Please address